tools/reviewer/local_server/service/CodeReviewService.java

```diff
@@ -376,6 +378,20 @@ public void ping(Empty req, StreamObserver<PongResponse> responseObserver) {
     responseObserver.onCompleted();
   }
 
+  @Override
+  public void removeWorkspace(
+      RemoveWorkspaceRequest req, StreamObserver<RemoveWorkspaceResponse> responseObserver) {
+    String workspacePath = getWorkspacePath(req.getWorkspaceName());
+
+    try {
+      fileUtils.deleteDirectory(workspacePath);
+      responseObserver.onNext(RemoveWorkspaceResponse.newBuilder().setRemoved(true).build());
+    } catch (IOException ignored) {
+      responseObserver.onNext(RemoveWorkspaceResponse.newBuilder().setRemoved(false).build());
```

Comment:
Maybe return a gRPC status code instead of a boolean?
E.g: `NOT_FOUND`
https://developers.google.com/maps-booking/reference/grpc-api/status_codes